pkgs/development/tools/gazou/default.nix

```diff
@@ -0,0 +1,28 @@
+# gazou.nix
+{ stdenv, fetchFromGitHub, cmake, pkgconfig , lib , tesseract , leptonica , qt5 }:
+
+stdenv.mkDerivation rec {
+  pname = "gazou";
+  version = "0.001-2023-08-18";
+
+  src = fetchFromGitHub {
+    owner = "kamui-fin";
+    repo = "gazou";
+    rev = "2114963698c94ec7e8f9023f8b6dbfa8437337de";
```

Comment:
Please use use `refs/tags/v0.2.1` here to match the version associated commit

Reply:
Addressed.